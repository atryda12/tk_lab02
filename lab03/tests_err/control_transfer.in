
int r=0;

string greet(string name)
{
    continue;
    return "Hey, " + name;
}

int inc(int n)
{
    break;
    return n+1;
}


return r;

print greet("world");
print inc(0);
