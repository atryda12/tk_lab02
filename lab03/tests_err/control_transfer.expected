
Error: continue instruction outside a loop: line 6
Error: break instruction outside a loop: line 12
Error: return instruction outside a function: line 17
